test-util/config.ts

```diff
@@ -52,18 +52,21 @@ export async function config(props?: NodeProps): Promise<Config> {
         ...props?.altRuntime ? [`--force-${props.altRuntime}`] : [],
       ],
       stderr: "piped",
-      stdout: "piped",
     });
+
     // For some reason, logs come in through `stderr`
     console.log(blue(`Piping node logs:`));
+
+    const encoder = new TextDecoder();
```

Comment:
The initialization cost of `TextDecoder`s is quite low. Imo not worth the overhead of a const definition.